__tests__/app.test.js

```diff
@@ -80,4 +80,57 @@ describe("Test /api/articles/:article_id", () => {
   });
 });
 
+describe("TEST /api/articles", () => {
+  test("GET 200 Should return an array of articles", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((article) => {
+          expect(typeof article).toBe("object");
+          expect(article.hasOwnProperty("author")).toBe(true);
+          expect(article.hasOwnProperty("title")).toBe(true);
+          expect(article.hasOwnProperty("article_id")).toBe(true);
+          expect(article.hasOwnProperty("topic")).toBe(true);
+          expect(article.hasOwnProperty("created_at")).toBe(true);
+          expect(article.hasOwnProperty("votes")).toBe(true);
+          expect(article.hasOwnProperty("article_img_url")).toBe(true);
+          expect(article.hasOwnProperty("comment_count")).toBe(true);
```

Comment:
big test! I'd have broken it down into multiple because this basically expects you to build the model in one go